airflow/configuration.py

```diff
@@ -29,6 +29,8 @@ def expand_env_var(env_var):
     `expandvars` and `expanduser` until interpolation stops having
     any effect.
     """
+    if not isinstance(env_var, str):
```

Comment:
May be better to use `basestr` over `str` here as it includes unicode strings and derivatives as well